views/templates/admin/configurations.tpl

```diff
@@ -17,18 +17,17 @@
 *}
 
 <link rel='stylesheet' type='text/css' href='//fonts.googleapis.com/css?family=Open+Sans:400,600&subset=latin,latin-ext'/>
-<link rel="stylesheet" type="text/css" href="{$mod_dir|escape:html:'UTF-8'}css/MyFontsWebfontsKit.css">
-<link rel="stylesheet" type="text/css" href="{$mod_dir|escape:html:'UTF-8'}css/configurations.css">
+<link rel="stylesheet" type="text/css" href="{$mod_dir|escape:html:'UTF-8'}views/css/configurations.css">
```

Comment:
With this new version, please move your CSS inclusions in addCss calls (Same example: https://github.com/PrestaShop/ganalytics/blob/master/ganalytics.php#L609-L625)